components/shapediver/appbuilder/AppBuilderImage.tsx

```diff
@@ -36,33 +41,43 @@ export function AppBuilderImageThemeProps(props: AppBuilderImageThemePropsType):
 }
 
 export default function AppBuilderImage(props: ImageNonStyleProps & ImageStyleProps & Props ) {
-	
-	const { anchor, target, ...rest } = props;
+	const { anchor, target, isSvg,  ...rest } = props;
 	const { radius, fit, withBorder, mah, maw } = useProps("AppBuilderImage", defaultStyleProps, rest);
 
 	const context = useContext(AppBuilderContainerContext);
 	const orientation = context.orientation;
 	const contain = fit === "contain";
+	const contentProps = {
+		...rest,
+		fit: fit,
+		radius: radius,
+		h: contain && orientation === "horizontal" ? "100%" : undefined,
+		w: contain && orientation === "vertical" ? "100%" : undefined,
+		className: withBorder ? classes.imgBorder : undefined,
+	};
 
-	const element = <Image
-		{...rest}
-		fit={fit}
-		radius={radius}
-		h={contain && orientation === "horizontal" ? "100%" : undefined}
-		w={contain && orientation === "vertical" ? "100%" : undefined}
-		mah={!contain && orientation === "horizontal" ? mah ?? "100%" : undefined}
-		maw={!contain && orientation === "vertical" ? maw ?? "100%" : undefined}
-		className={withBorder ? classes.imgBorder : undefined}
-	/>;
-
-	const elementWithAnchor = anchor ? <Anchor 
-		href={anchor} 
-		target={target}
-		display="contents"
-	>
-		{element}
-	</Anchor> : element;
-
-	return elementWithAnchor;
+	const element = isSvg ? (
+		<Svg
+			{...contentProps}
+			srcUrl={rest.src}
+			mah={mah}
+			maw={maw}
```

Comment:
@LemToUp did you not use the same logic for `mah` and `maw` on purpose? 
I mean this one: 

```
mah={!contain && orientation === "horizontal" ? mah ?? "100%" : undefined}
maw={!contain && orientation === "vertical" ? maw ?? "100%" : undefined}
```

Reply:
We must make the layout behavior exactly the same regardless of whether isSvg is true or false.

Reply:
When removing the maw property from the json file, I get this: 

<img width="600" alt="image" src="https://github.com/user-attachments/assets/51b14119-76f0-468d-b36d-4876fb743a96" />

Probably that's why you added maw, but we must make it work without that.